morgoth/base/static/js/reducers/addonsReducer.js

```diff
@@ -12,8 +12,6 @@ const INITIAL_STATE = {
 };
 
 export default (state = INITIAL_STATE, action) => {
-  let error;
```

Comment:
By the way, to deal with variables in reducers, I found this pattern to work well:

```javascript
export default (state = INITIAL_STATE, action) => {
  switch (action.type):
    case FETCH_ADDONS_FAILURE: {
      let error = action.payload;
      return {
        ..state,
        addonsList: {
          error,
          loading: false,
        };
    }
}
```